### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by rijnhard** [intacct/intacct-sdk-php#201]
Hi @tim-miller-sage @karstenintacct 

Any chance we can get a new release now that the changes for PHP 8.2 have been merged?

**Comment by dominicbvm** [comment]
Yes, a new release would be great for ease to pull in with composer.

**Comment by bosam** [comment]
I confirm that it would be greatly appreciated!

Currently the psr/log ^1 is locking a lot of dependencies from being upgraded.

**Comment by rijnhard** [comment]
@tim-miller-sage even though this is deprecated, can we still get a v3.30 release for what has already been merged?
